wallet-services/src/main/resources/locale/portlet/Portlets_en.properties

```diff
@@ -72,5 +73,7 @@ exoplatform.wallet.error.wrongPrivateKey=Private key doesn't match address {0}
 exoplatform.wallet.error.errorImportingPrivateKey=Error importing private key
 exoplatform.wallet.error.errorProcessingNewKeys=Error processing new keys
 exoplatform.wallet.error.walletNotFound=Can't find your wallet
+
 exoplatform.wallet.metamask.welcomeMessage=Welcome to digital workplace \n\nClick to SignIn and accept digital workplace terms. This actions will not trigger a blockchain transaction and or any Gas fees. \n\nWallet address: \n{0} \n\nNonce: \n{1}
 exoplatform.wallet.metamask.message.connectedSuccess=Wallet connected to Metamask successfully !
+exoplatform.wallet.metamask.message.transactionSent=Tranasction sent 
```

Comment:
add this to Wallet_en.properties please.

Reply:
done